### PR TITLE
Ricochet: add verion 1.1.4

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/ricochet.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.1.4",
+    "description": "Anonymous peer-to-peer instant messaging.",
+    "homepage": "https://github.com/ricochet-im/ricochet",
```
Comment (Ash258) [inline bucket/ricochet.json]: https://ricochet.im

File: bucket/ricochet.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "innosetup": true,
+    "bin": "ricochet.exe",
+    "shortcuts": [
+        [
+            "ricochet.exe",
+            "Ricochet"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\tor.exe\" -Force"
+}
```
Comment (Ash258) [inline bucket/ricochet.json]: No checkver and autoupdate

File: bucket/ricochet.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "depends": "tor",
+    "innosetup": true,
+    "bin": "ricochet.exe",
+    "shortcuts": [
+        [
+            "ricochet.exe",
+            "Ricochet"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\tor.exe\" -Force"
```
Comment (Ash258) [inline bucket/ricochet.json]: Why do you remove this tor execuable and depending on tor manifest?

File: bucket/ricochet.json (hunk 2 above)
Comment (linsui) [reply bucket/ricochet.json]: It has stoped for more than 2 years... Should I add checkver anf autoupdate?

File: bucket/ricochet.json (hunk 3 above)
Comment (linsui) [reply bucket/ricochet.json]: The bundled tor is outdate and unsafe.

File: bucket/ricochet.json (hunk 1 above)
Comment (linsui) [reply bucket/ricochet.json]: It will redirect to github.

Comment (Ash258) [reply bucket/ricochet.json]: I can without problem my VMs physically running in Czechia, Romania, Germany, Switzerland, England.

Problem is on your end.

File: bucket/ricochet.json (hunk 2 above)
Comment (Ash258) [reply bucket/ricochet.json]: There is no reason to not add it. Repository is not archived, no official statement, anything, ...

File: bucket/ricochet.json (hunk 1 above)
Comment (linsui) [reply bucket/ricochet.json]: Thanks, you are right, it's blocked in China.

File: bucket/ricochet.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.1.4",
+    "description": "Anonymous peer-to-peer instant messaging.",
+    "homepage": "https://ricochet.im",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/ricochet-im/ricochet/releases/download/v1.1.4/ricochet-1.1.4-win-install.exe#/dl.7z",
```
Comment (Ash258) [inline bucket/ricochet.json]: Why this rename, when you are using `innosetup`??

Comment (linsui) [reply bucket/ricochet.json]: Sorry, my fault...

File: bucket/ricochet.json (hunk 3 above)
Comment (Ash258) [reply bucket/ricochet.json]: 1. Will this work with new versions of tor?
1. Will this work when there is no file `tor.exe` in directory, but in Path

Comment (linsui) [reply bucket/ricochet.json]: Yeah, it works.

**Pull request merged**